Sources/SwiftToolchain/Toolchain.swift

```diff
@@ -223,10 +223,10 @@ public final class Toolchain {
     let builderArguments = try [
       swiftPath.pathString, "build", "-c", isRelease ? "release" : "debug", "--product", product,
       "--enable-test-discovery", "--destination", destination ?? inferDestinationPath().pathString,
-      "-Xswiftc", "-color-diagnostics",
+//      "-Xswiftc", "-color-diagnostics",
     ]
 
-    try ProcessRunner(builderArguments, terminal).waitUntilFinished()
+    try ProcessRunner(builderArguments, loadingMessage: "Compiling...", terminal).waitUntilFinished()
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/SwiftToolchain/Toolchain.swift;
  Line: 229;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 101 characters (`line_length`)